episodes/quality-control-and-manual-segmentation.md

```diff
@@ -0,0 +1,547 @@
+controls). For example, if your active label is 2, then clicking on a nucleus 
+with label 1 will change it to 2. You can see more details about the fill bucket 
+in [Napari's documentation
+](https://napari.org/stable/howtos/layers/labels.html#using-the-fill-bucket)
+
+### ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/picker.svg
+){alt="A screenshot of Napari's picker button" height='30px'} icon
+
+This icon activates the 'colour picker'. This allows you to click on a pixel in 
+the viewer and immediately select the corresponding label. This is especially 
+useful when you have hundreds of labels, which can make them difficult to keep 
+track of with the 'label:' option only.
+
+### Contour
+
+The contour setting controls how labels are displayed. By default (when 0), 
+labelled pixels are displayed as solid colours. Increasing the contour value to 
+one or higher, will change this to show the outline of labelled regions instead. 
+Higher values will result in a thicker outline. This can be useful if you don't 
+want to obscure the underlying image while viewing your segmentation.
+
+### n edit dim
+
+This controls the number of dimensions that are used for editing labels. By 
+default, this is 2 so all painting and editing of the `Labels` layer will only 
+occur in 2D. If you want to paint/edit in 3D you can increase this to 3. Now 
+painting will not only affect the currently visible image slice, but slices 
+above and below it also. Be careful with the size of your brush when using this 
+option!
+
+### preserve labels
+
+Activating 'preserve labels' will ensure that any painting/erasing you do will 
+only affect your currently active label. This is especially useful if you need 
+to paint very close to existing labels without affecting them.
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Manual vs automated?
+
+Now that we've done some manual segmentation in Napari, you can probably tell 
+that this is a very slow and time consuming process (especially if you want to 
+segment in full 3D!). Therefore, it's generally only suitable for small images 
+in small quantities. If we want to efficiently scale to larger images in large 
+quantities, then we need more automated methods. We'll look at some of these in 
+the next few episodes. It's worth bearing in mind though that automated methods 
+are rarely perfect, so it's very likely that you will have to do some manual 
+segmentation cleanup from time to time.
+
+::::::::::::::::::::::::::::::::::::: keypoints 
+
+- The first step in any image processing workflow is quality control. For 
+example, checking the histograms of your images.
+
+- Segmentation is the process of identifying what each pixel in an image 
+represents e.g. is that pixel part of a nucleus or not?
+
+- Segmentation can be broadly split into 'semantic segmentation' and 'instance 
```

Comment:
I would add brief descriptions either in this keypoint or as separate additional keypoints about what these types of segmentations are!